### ISSUE TITLE
 support RTX 3080 12GB LHR

### ISSUE THREAD

**Issue by rayven96** [trexminer/T-Rex#996]
please add support for 3080 12gb LHR

**Comment by rayven96** [comment]
as I see from the first reviews, rtx 3080 12gb is not yet possible to implement in the -lhr tune 74 mode

**Comment by jbrooks84** [comment]
Same, only getting like 62MH/s with 3080 12gb LHR

**Comment by jbrown228** [comment]
Please, I’m only getting 59mhs

**Comment by pedroleles95** [comment]
Any update if the 12gb is going to be minable with 74% LHR unlock ?

**Comment by marcioaneves** [comment]
I have the same problem - Colorful 3080 TI - 12 Gb

**Comment by chakumon** [comment]
Hi, please enable support for this too, just received my gpu 😎

**Comment by marcioaneves** [comment]
> Hi, please enable support for this too, just received my gpu 😎

Do you have hashrate problem as well ?

**Comment by chakumon** [comment]
> > Hi, please enable support for this too, just received my gpu 😎
> 
> Do you have hashrate problem as well ?

Yeah getting 61MH, trex not detecting the card as LHR..

**Comment by jbrown228** [comment]
> > > Hi, please enable support for this too, just received my gpu 😎
> > 
> > 
> > Do you have hashrate problem as well ?
> 
> Yeah getting 61MH, trex not detecting the card as LHR..

Try 1260 core 2000 mem, Linux

**Comment by chakumon** [comment]
> > > > Hi, please enable support for this too, just received my gpu 😎
> > > 
> > > 
> > > Do you have hashrate problem as well ?
> > 
> > 
> > Yeah getting 61MH, trex not detecting the card as LHR..
> 
> Try 1260 core 2000 mem, Linux

I’m already at 1400 core 2800 mem.. the issue is not the overclocks.. it’s the miner..

**Comment by jbrown228** [comment]
> > > > > Hi, please enable support for this too, just received my gpu 😎
> > > > 
> > > > 
> > > > Do you have hashrate problem as well ?
> > > 
> > > 
> > > Yeah getting 61MH, trex not detecting the card as LHR..
> > 
> > 
> > Try 1260 core 2000 mem, Linux
> 
> I’m already at 1400 core 2800 mem.. the issue is not the overclocks.. it’s the miner..

Yeah I know, but I got stable 63.5 mhs with those lower settings, I had mine set higher before, just trying to help

**Comment by marcioaneves** [comment]
> > > > > > Hi, please enable support for this too, just received my gpu 😎
> > > > > 
> > > > > 
> > > > > Do you have hashrate problem as well ?
> > > > 
> > > > 
> > > > Yeah getting 61MH, trex not detecting the card as LHR..
> > > 
> > > 
> > > Try 1260 core 2000 mem, Linux
> > 
> > 
> > I’m already at 1400 core 2800 mem.. the issue is not the overclocks.. it’s the miner..
> 
> Yeah I know, but I got stable 63.5 mhs with those lower settings, I had mine set higher before, just trying to help

I'm already at 1200 core 200 mem.  I got stable ~69 mhs.

**Comment by jbrown228** [comment]
> > > > > > > Hi, please enable support for this too, just received my gpu 😎
> > > > > > 
> > > > > > 
> > > > > > Do you have hashrate problem as well ?
> > > > > 
> > > > > 
> > > > > Yeah getting 61MH, trex not detecting the card as LHR..
> > > > 
> > > > 
> > > > Try 1260 core 2000 mem, Linux
> > > 
> > > 
> > > I’m already at 1400 core 2800 mem.. the issue is not the overclocks.. it’s the miner..
> > 
> > 
> > Yeah I know, but I got stable 63.5 mhs with those lower settings, I had mine set higher before, just trying to help
> 
> I'm already at 1200 core 200 mem. I got stable ~69 mhs.

You stated you have a 3080ti, you should be getting like 90mhs, don’t use these settings

**Comment by fungwowraider** [comment]
Is there any update on this?  I recently purchased 3080 12GB and it's awful for mining it seems.  Trex miner is not detecting as LHR and forcing LHR is even worse.  Best settings I can find so far is getting ~60 mhs @ 280kH/W

**Comment by chakumon** [comment]
> > > > > > Hi, please enable support for this too, just received my gpu 😎
> > > > > 
> > > > > 
> > > > > Do you have hashrate problem as well ?
> > > > 
> > > > 
> > > > Yeah getting 61MH, trex not detecting the card as LHR..
> > > 
> > > 
> > > Try 1260 core 2000 mem, Linux
> > 
> > 
> > I’m already at 1400 core 2800 mem.. the issue is not the overclocks.. it’s the miner..
> 
> Yeah I know, but I got stable 63.5 mhs with those lower settings, I had mine set higher before, just trying to help

Hey dude! Surprisingly it did increase to 63MH! I’m at 1200 core 2000 mem pulling 210w! Thanks man.. just hope the devs are doing something about this.. appreciate it!

**Comment by R3LIAS1** [comment]
We need new version of trex. I have 3080 and I have only 63MH/s 198watt

**Comment by jbrown228** [comment]
So trex 25.2 unlocks the LHR for 3080 12gb, but the target hashrate is lower than what I was getting before ~55mhs, going back to 24.8 for now

**Comment by fungwowraider** [comment]
> 

What's your OC settings?  I've been struggling to get above 300 kH/W.   Currently getting 61MH/s at 215W.

Power level=54, Core=0, Memory=850

**Comment by chakumon** [comment]
> What's your OC settings? I've been struggling to get above 300 kH/W. Currently getting 61MH/s at 215W.
> 
> Power level=54, Core=0, Memory=850

In windows trex batch file
—lock-cclock 1200 —mclock 1000

**Comment by fungwowraider** [comment]
There's a new update 25.2.  Anyone know if that fixed this?

**Comment by chakumon** [comment]
> There's a new update 25.2. Anyone know if that fixed this?

No mention of 3080 12GB.. i guess they need to rework abit as the driver is also different than other LHR cards..

**Comment by jbrown228** [comment]
> > There's a new update 25.2. Anyone know if that fixed this?
> 
> No mention of 3080 12GB.. i guess they need to rework abit as the driver is also different than other LHR cards..

It actually shows as LHR with the new update, but the best hashrate I could get was 55, I turned the lhr tune off

**Comment by chakumon** [comment]
> > > There's a new update 25.2. Anyone know if that fixed this?
> > 
> > 
> > No mention of 3080 12GB.. i guess they need to rework abit as the driver is also different than other LHR cards..
> 
> It actually shows as LHR with the new update, but the best hashrate I could get was 55, I turned the lhr tune off for just the 3080 12gb

Yeah i noticed that too but the lhr keeps kicking in.. switched to teamblackminer instead.. getting 65MH..

**Comment by fungwowraider** [comment]
I had it higher, around 1.5 on blake3 but not worth the power for me right now.  The real issue is eth algo on this new 3080 12GB.  Something is different compared to all other lhr cards I have

**Comment by mwpastore** [comment]
@fungwowraider Yeah, I hear you. Mine's keeping a room warm, and electricity is cheaper than gas right now, so it's worth it for me. 

Agreed. I'd much prefer a "true" 74% unlock mode for this card. I'm using lhr-tune 0 for mine as well.

**Comment by yilmazburk** [comment]
I have also same problem, here ASUS Rog Strix 3080 12GB LHR, miner can not recognize LHR 
Is there any plan to fix this issue? If not, then i should sell my card :(

**Comment by akifhatipoglu** [comment]
Yes, I have the same. I have two ASUS Rog Strix 3080 12GB. it can not recognize LHR. Will there be any development?

**Comment by Barsik27** [comment]
the same, GIGABYTE RTX3080 12GB. repeated LHR locks\unlock - LHR AUTOTUNE LEVEL not work
gminer 2.91 LHR 57.9% works fine 67Mhs only with mc:+550 pl:80-85% cc:-200
set LHR to 57.9% in t-rex   "lhr_tune": "-1,-1,-1,-1,57.9" - works fine 67.54 mhs

**Comment by Coddo-Python** [comment]
Any updates?

**Comment by mwpastore** [comment]
https://www.youtube.com/watch?v=r0tru4eu92M&t=806s

**Comment by Coddo-Python** [comment]
> [Here](https://www.youtube.com/watch?v=r0tru4eu92M&t=806s)'s the developer of lolMiner explaining how the 3080 12GB is different and why the LHR unlock doesn't work.

Yes, I am aware of this, but NBMiner just released a 90% unlock for the 3080 12gb and 3050! So why can't t-rex?

**Comment by mwpastore** [comment]
@Coddo-Python NBMiner 41.3 (with the 90% unlock) was released six days ago, so we just have to be patient!
